### PR TITLE
WIP: Improve documentation on how to contribute to documentation

### PR DESCRIPTION
**Pull request by SJaffa**
At RSECon24 R dev day. Looking at https://github.com/r-devel/r-dev-day/issues/55, and https://github.com/r-devel/rdevguide/issues/118, but not sure if it fully closes either of them yet.

New contributors can often get started by updating or improving documentation. Add more details to the R dev day guide about how to do this. 

**Changes made**:
I have added a note to the introduction page and reordered the documentation page to help people find the different kinds of documentation they can work on. 

**Not done**:
Also mentioned in #118 is adding some examples of documentation issues, perhaps linking to a successfully closed issue.I have not done this as I run out of time, so someone else will need to take this on.

### CODE REVIEW

File: 01-Introduction.Rmd [hunk 1]
```diff
@@ -9,8 +9,8 @@ Contributions to base R are possible in a number of different ways. Some of them
 1. Contributing to bug fixing: Refer [Bug Tracking](#BugTrack) and [Reviewing Bugs](#ReviewBugs).
 2. Contributing to translations: Refer [Translations](#).
 3. Testing R before release: Refer [Testing Pre-release R Versions](#TestRVer).
+4. Contributing to documentation and tutorials: Refer [Documenting R](#Doc).
```
Comment (SJaffa) [inline 01-Introduction.Rmd:12]: Is it worth adding a note here that this is a common starting point for new contributors, or add this later?

File: 01-Introduction.Rmd [hunk 2]
```diff
@@ -9,11 +9,13 @@ Contributions to base R are possible in a number of different ways. Some of them
 1. Contributing to bug fixing: Refer [Bug Tracking](#BugTrack) and [Reviewing Bugs](#ReviewBugs).
 2. Contributing to translations: Refer [Translations](#).
 3. Testing R before release: Refer [Testing Pre-release R Versions](#TestRVer).
+4. Contributing to documentation and tutorials, including this guide: Refer [Documenting R](#Doc) and [Contributing to this guide](#how-to-contribute-to-this-guide).
 
-<!-- TODO(Saranjeet): Refer Contributing to R tutorial -->
 <!-- TODO(Saranjeet): Welcome to Bug BBQ slides -->
 <!-- TODO(Saranjeet): README on Bug BBQ repo -->
 
+Maintainers and contributors are requested to follow this project's [code of conduct](https://github.com/r-devel/rdevguide/blob/main/CONDUCT.md).
```
Comment (SJaffa) [inline 01-Introduction.Rmd:17]: This change is not directly related to my aim of helping people contribute to docs, but in reading as a new contributor I found I clicked away on links before getting to this important statement at the end of the page, so it may be better to put it higher up to ensure people are aware fo the CoC before they go into the details of how to contribute.

File: 08-documenting.Rmd [hunk 3]
```diff
@@ -4,7 +4,50 @@ R has a substantial body of documentation, comprising help files for the core pa
...
+
+Maintaining the accuracy of R's documentations and keeping a high level of quality takes a lot of effort. Community members, like you, help with writing, editing, and updating content, and these contributions are appreciated and welcomed.
+
+Looking at pre-existing documentation source files can be very helpful when getting started.
+
+You can directly search for [documentation issues/bugs on Bugzilla](https://bugs.r-project.org/buglist.cgi?component=Documentation&list_id=22501&product=R&resolution=---). Issues vary from typos to unclear documentation and items lacking documentation.
+
+If you see a documentation issue that you would like to tackle, you can leave a comment on the issue saying you are going to try to solve the issue and mention roughly how long you think you will take to do so (this allows others to take on the issue if you happen to forget or lose interest).
+
+If you find some typo or problem on [CRAN](https://cran.r-project.org) after checking the problem or typo you can write a polite email to `cran-sysadmin@r-project.org` and one of the R-core members working with CRAN. You'll probably get a prompt reply about how the issue is going to be fixed.
```
Comment (llrs) [inline 08-documenting.Rmd:19]: In line 26 (next paragraph) the email suggested is different. I think CRAN@r-project.org is the right one.

**Review by llrs — COMMENTED** [review]
Nice addition, I hadn't had time to check it all well, but I think there is some duplication on a paragraph (see the comment)

**Comment by cjrace** [comment]
I've continued this PR in my own fork and raised #188, in there I've copied all the same changes across and responded to all comments raised here. I think this PR can be closed now?

**Comment by hturner** [comment]
Thanks @SJaffa for making a start here. As @cjrace says, he has taken your suggestion on board and we are working on a new PR (#188) that will enable us to close the issue. We'll be sure to acknowledge your contribution when that gets merged in!

**Comment by hturner** [comment]
Closed by #188.

@all-contributors please add @SJaffa for content and documentation.
